tests/CiIntegrationTest.php

```diff
@@ -168,6 +163,80 @@ public function testIntegration()
         $this->assertSame(0, $result3['code']);
     }
 
+    public function testIntegrationWithChangesToConfigFile()
+    {
+        static::executeCommand(implode(' && ', array(
+            'git checkout -b changes-to-config-file -q',
+            'echo "" >> dir\ b/file\ b.php',
+            'echo "echo 1;" >> dir\ b/file\ b.php',
+            // `sed -i ...` is not handled the same on Linux and macOS
+            'sed -e \'s/@Symfony/@PSR2/\' .php_cs.dist > .php_cs.dist.new',
```

Comment:
you have test with `.php_cs.dist` modified.
you shall add test cases about `.php_cs` and `composer.lock`